styles.css

```diff
@@ -243,7 +251,7 @@ body {
 .timeline-number {
   width: 40px;
   height: 40px;
-  background-color: var(--primary);
+  background-color: var(--accent);
```

Comment:
I wanted to have this as grey. Could you please change this to the variable with #586069